appeals/web/src/server/lib/display-page-formatter.js

```diff
@@ -244,9 +238,14 @@ const formatDocumentValuesAsList = ({ appealId, documents, isAdditionalDocuments
 			});
 		}
 
-		if (htmlProperty.pageComponents.length > 0) {
+		if (htmlProperty.pageComponents.length > 1) {
 			htmlProperty.wrapperHtml = {
-				opening: '<ul class="govuk-list">',
+				opening: '<ol class="govuk-list govuk-list--number pins-file-list">',
+				closing: '</ol>'
+			};
+		} else if (htmlProperty.pageComponents.length > 0) {
+			htmlProperty.wrapperHtml = {
+				opening: '<ul class="govuk-list pins-file-list">',
 				closing: '</ul>'
 			};
 		}
```

Comment:
Is this just so we don't have 1 element numbered 1?

Reply:
Yes exactly that, the ticket has an AC to not display a number if there's only one item